demos/golang/go_sqlite/run_go_sqlite_demo.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -e
+
+BLUE='\033[1;34m'
+NC='\033[0m'
+
+# Install SQLite with occlum-go
+occlum-go get -u -v github.com/mattn/go-sqlite3
+
+# Build the Golang SQLite demo program using the Occlum Golang toolchain(i.e., occlum-go)
+occlum-go build -o simple_demo simple_demo.go
+
+sqlite_demo="./simple_demo"
+if [ ! -f $sqlite_demo ];then
+    echo "Error: cannot stat file '$sqlite_demo'"
+    echo "Please build the Golang SQLite demo program using occlum-go"
+    exit 1
+fi
+
+# Init Occlum Workspace
+rm -rf simple_demo_instance && mkdir simple_demo_instance
+cd simple_demo_instance
+occlum init
+new_json="$(jq '.resource_limits.user_space_size = "2560MB" |
+	.resource_limits.kernel_space_heap_size="320MB" |
+	.resource_limits.kernel_space_stack_size="10MB" |
+	.process.default_stack_size = "40MB" |
+	.process.default_heap_size = "320MB" |
+	.process.default_mmap_size = "960MB" ' Occlum.json)" && \
+echo "${new_json}" > Occlum.json
+
+# Copy program into Occlum Workspace and build
+cp ../simple_demo image/bin
+occlum build
+
+# Run the hello world sample
```

Comment:
`Run the hello world sample` should be corrected.

Reply:
Done. Thanks.